test/e2e/reaper_test.go

```diff
@@ -253,10 +261,15 @@ func testRemoveReaperFromK8ssandraCluster(
 	checkReaperK8cStatusReady(t, f, ctx, kcKey, dcKey)
 }
 
-func testReaperApi(t *testing.T, ctx context.Context, k8sContextIdx int, clusterName, keyspace string) {
+func testReaperApi(t *testing.T, ctx context.Context, k8sContextIdx int, clusterName, keyspace string, f *framework.E2eFramework, namespace string, authEnabled bool) {
```

Comment:
Minor: could you keep the same order of parameters used in other functions? that is,

```
t *testing.T, 
f *framework.E2eFramework, 
ctx context.Context, 
k8sContextIdx int,
namespace string, 
clusterName string,
<others>
```